### DISCUSSION TITLE
Analytics only shows IOS devices

### DISCUSSION THREAD

**Discussion by jongbonga** [capawesome-team/capacitor-firebase#122] [Q&A]
I just want to confirm that I don't have to add the `$firebaseAnalyticsVersion` manually on the variables.gradle file as per [example](https://github.com/robingenz/capacitor-firebase-plugin-demo/blob/main/android/variables.gradle).

I'm getting analytics for ios devices but no Androids

**Comment by robingenz** [comment]
You do not have to set the `$firebaseAnalyticsVersion` variable in `variables.gradle`. This is optional to use another version than the default version.
Please create an issue with a reproducible, minimal example.

**Reply by jongbonga** [reply]
It's tricky since my project is big and I can't scrap it up to have a minimal version, and a from-scratch will probably not reproduce the issue.
But I noticed the build.gradle file has

```
try {
    def servicesJSON = file('google-services.json')
    if (servicesJSON.text) {
        apply plugin: 'com.google.gms.google-services'
        apply plugin: 'com.google.firebase.firebase-perf' // Firebase Performance Monitoring plugin
        apply plugin: 'com.google.firebase.crashlytics' // Firebase Crashlytics plugin
    }
} catch(Exception e) {
    logger.warn("google-services.json not found, google-services plugin not applied. Push Notifications won't work")
}
```

Mine doesn't contain the line `apply plugin: 'com.google.firebase.firebase-perf'`. Could that be the problem?

**Reply by robingenz** [reply]
No, that line is only necessary for `@capacitor-firebase/performance`.

> and a from-scratch will probably not reproduce the issue.

if you can't reproduce it from scratch, then it's likely that it's not a bug in the plugin but a problem with your app. Unfortunately I can only search for a solution when I understand the problem.

**Reply by jongbonga** [reply]
So there is no setup needed for Android?

**Reply by robingenz** [reply]
No, just install the plugin. Have you tried the Demo App: https://github.com/robingenz/capacitor-firebase-plugin-demo? Does it work there for you?
